graphql/util.go

```diff
@@ -408,6 +408,14 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 			return errors.Wrap(err, "Error setting patch variants and tasks"), http.StatusInternalServerError, "", ""
 		}
 
+		projectRef, err := model.FindOneProjectRef(project.Identifier)
```

Comment:
need to check if project ref is nil also (although we also do this in the `if` case, so probably we should just factor this out so it can be used in both places rather than repeating code)